corehq/apps/app_manager/suite_xml/features/case_tiles.py

```diff
@@ -200,6 +201,10 @@ def _get_xpath_function(self, column):
         else:
             xpath_function = self._escape_xpath_function(get_column_generator(
                 self.app, self.module, self.detail, column).xpath_function)
+        if column.format == "translatable-enum":
+            for enum in column.enum:
+                if enum.key in xpath_function and 'k' + enum.key not in xpath_function:
```

Comment:
Can you explain the 'k'? I am confused.

Reply:
when adding the enum keys to the app_strings file, it always uses [key_as_variable](https://github.com/dimagi/commcare-hq/blob/d645bca7d995e7830c1ca5836cb4c63457bca8d1/corehq/apps/app_manager/models.py#L979) which prepends a `k` to the variable name so it won't start with a numeral/be a valid xml variable. I don't want the user to always have to add `k` when referencing the variable so I'm correcting it in the backend instead so that a calculation like `concat($test)` is actually read as `concat($ktest)` which maps to a valid string in the app_strings file. 

Looking at it again... I definitely could've just made it so it doesn't prepend `k` for translatable-text formats but I guess it's nice that you could potentially start a key name with a number and you could reuse keys you use for custom variables without it overlapping.